_dev/apps/ui/src/components/enhanced-conversions/alert-sign-gads-tos.vue

```diff
@@ -32,18 +35,22 @@
 
 <script lang="ts">
 import {defineComponent} from 'vue';
-import {mapGetters} from 'vuex';
-import GettersTypesApp from '@/store/modules/app/getters-types';
 import googleUrl from '@/assets/json/googleUrl.json';
 
 export default defineComponent({
   computed: {
-    ...mapGetters('app', [
-      GettersTypesApp.GET_FEATURE_FLAG_ENHANCED_CONVERSIONS,
-    ]),
     linkToTermsOfServices(): string {
       return googleUrl.googleAdsConversionsTermsAndConditions;
     },
+    linkToInstructions(): string {
+      // TODO: Replace with actual URL
+      return 'perdu.com';
```

Comment:
Should you replace it or not ?

Reply:
Of course, I'm waiting for the actual URL